com.unity.multiplayer.mlapi/Editor/NetworkedAnimatorEditor.cs

```diff
@@ -9,92 +9,70 @@ namespace UnityEditor
     [CanEditMultipleObjects]
     public class NetworkAnimatorEditor : Editor
     {
-        // TODO @mfatihmar (Unity): Re-implement this after `NetworkedAnimator` re-implementation
-        /*
-        private NetworkedAnimator networkedAnimatorTarget;
+        private NetworkedAnimator m_Target;
+
         [NonSerialized]
-        private bool initialized;
+        private bool m_Initialized;
 
-        private SerializedProperty animatorProperty;
-        private GUIContent animatorLabel;
+        private SerializedProperty m_AnimatorProperty;
+        private GUIContent m_AnimatorLabel;
 
-        void Init()
+        private void Initialize()
         {
-            if (initialized)
-                return;
-
-            initialized = true;
-            networkedAnimatorTarget = target as NetworkedAnimator;
+            if (m_Initialized) return;
 
-            animatorProperty = serializedObject.FindProperty("_animator");
-            animatorLabel = new GUIContent("Animator", "The Animator component to synchronize.");
-        }
+            m_Initialized = true;
+            m_Target = target as NetworkedAnimator;
 
-        public override void OnInspectorGUI()
-        {
-            Init();
-            serializedObject.Update();
-            DrawControls();
-            serializedObject.ApplyModifiedProperties();
+            m_AnimatorProperty = serializedObject.FindProperty("m_Animator");
+            m_AnimatorLabel = new GUIContent("Animator", "The Animator component to synchronize.");
         }
 
-        void DrawControls()
+        private void DrawControls()
         {
             EditorGUI.BeginChangeCheck();
-            EditorGUILayout.PropertyField(animatorProperty, animatorLabel);
+            EditorGUILayout.PropertyField(m_AnimatorProperty, m_AnimatorLabel);
+            if (EditorGUI.EndChangeCheck()) m_Target.ResetTrackedParams();
 
-            if (EditorGUI.EndChangeCheck())
-                networkedAnimatorTarget.ResetParameterOptions();
+            var animator = m_Target.animator;
+            if (ReferenceEquals(animator, null)) return;
 
-            if (networkedAnimatorTarget.animator == null)
-                return;
+            var animatorController = animator.runtimeAnimatorController as AnimatorController;
+            if (ReferenceEquals(animatorController, null)) return;
 
-            var controller = networkedAnimatorTarget.animator.runtimeAnimatorController as AnimatorController;
-            if (controller != null)
+            EditorGUI.indentLevel += 1;
+            var showWarning = false;
             {
-                var showWarning = false;
-                EditorGUI.indentLevel += 1;
-                int i = 0;
-
-                foreach (var p in controller.parameters)
+                int paramIndex = 0;
+                foreach (var animParam in animatorController.parameters)
                 {
-                    if (i >= 32)
+                    if (paramIndex >= 32)
```

Comment:
Magical

Reply:
yeah, already was :(
_(probably comes from 32-bits integer and bitwise flags logic)_